features/bevragen/persoon/naam/autorisatie/autorisatie.feature

```diff
@@ -0,0 +1,110 @@
+
+      Voorbeelden:
+      | fields                                    | ad hoc rubrieken                            | missende autorisatie |
+      | naam.voornamen                            | 10120 10220 10230 10240 16110 PANM01 PANM02 | 10210                |
+      | naam.adellijkeTitelPredicaat              | 10120 10210 10230 10240 16110 PANM01 PANM02 | 10220                |
+      | naam.adellijkeTitelPredicaat.code         | 10120 10210 10230 10240 16110 PANM01 PANM02 | 10220                |
+      | naam.adellijkeTitelPredicaat.omschrijving | 10120 10210 10230 10240 16110 PANM01 PANM02 | 10220                |
+      | naam.voorvoegsel                          | 10120 10210 10220 10240 16110 PANM01 PANM02 | 10230                |
+      | naam.geslachtsnaam                        | 10120 10210 10220 10230 16110 PANM01 PANM02 | 10240                |
+      | naam.aanduidingNaamgebruik                | 10120 10210 10220 10230 10240 PANM01 PANM02 | 16110                |
+      | naam.voorletters                          | 10120 10210 10220 10230 10240 16110 PANM02  | PANM01               |
+      | naam.volledigeNaam                        | 10120 10210 10220 10230 10240 16110 PANM01  | PANM02               |
+
+    @fout-case
+    Abstract Scenario: Afnemer vraagt om hele groep naam en is niet geautoriseerd voor <missende autorisatie>
+      Gegeven de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | <ad hoc rubrieken>              | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 000008 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000024                       |
+      | fields              | naam                            |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                                  |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
+      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
+      | status   | 403                                                                     |
+      | code     | unauthorizedField                                                       |
+      | instance | /haalcentraal/api/brp/personen                                          |
+
+      Voorbeelden:
+      | ad hoc rubrieken                                  | missende autorisatie |
+      | 10120 10220 10230 10240 10410 16110 PANM01 PANM02 | 10210                |
```

Comment:
Bij deze scenarios wordt aangegeven dat 10410 (geslacht) een vereiste rubriek is voor vragen van field naam. In de bovenstaande scenarios komt dat rubriek niet voor. Klopt dit?

Reply:
het voorbeeld op regel 107 met ontbreken van 10410 heb ik verwijderd. die was voorheen verplicht voor het samenstellen van volledigeNaam, maar is nu juist niet meer relevant.